css-positioning/absolute.css

```diff
@@ -0,0 +1,47 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+
+  /* your code here */
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  bottom: 78%;
+  left: 80%;
```

Comment:
This is a _big_ movement. Instead of moving it all the way from the other side, you should try moving from the `right` and `top`